alws/alembic/versions/a3f6778d2499_added_builted_srpm_url.py

```diff
@@ -0,0 +1,28 @@
+"""added_builted_srpm_url
+
+Revision ID: a3f6778d2499
+Revises: e88b665182b9
+Create Date: 2021-12-26 16:47:30.823966
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'a3f6778d2499'
+down_revision = 'e88b665182b9'
```

Comment:
I believe it would be changed in the future

Reply:
Fixed